### PR TITLE
New Layout for Debug Text if more then 1 player

### PR DESCRIPTION
**Pull request by nevespt**
I added the code to change the Debug Text Position, size and anchor if more then 1 player is present, else stays as before

### CODE REVIEW

**Comment by nevespt** [comment]
**Exemple In-Game**
1 Player (Default):
image

More then 1 Player (New):
image

File: Assets/Script/Gameplay/GameManager.cs [hunk 1]
```diff
@@ -134,6 +134,19 @@ private void Awake()
...
+            if (_yargPlayers.Count > 1)
+            {
+                _debugText.alignment = TextAlignmentOptions.TopLeft;
+                _debugText.fontSize = 25;
+                _debugText.font = Resources.Load<TMP_FontAsset>("Fonts & Materials/Barlow SDF");
+                _debugText.rectTransform.anchorMin = Vector2.up;
+                _debugText.rectTransform.anchorMax = Vector2.up;
+                _debugText.rectTransform.pivot = Vector2.up;
+                _debugText.rectTransform.anchoredPosition = new Vector2(25, -60);
+            }
```
Comment (EliteAsian123) [inline Assets/Script/Gameplay/GameManager.cs:147]: I'm not sure if this whole part is necessary; these kinds of things should be configured in the inspector instead, and moving the debug test based on position is a bit of an over complication, especially for *debug* text. The user isn't really supposed to see it anyway. If you just move the text to the top left in the inspector, and commit that, I'll merge this.

File: Assets/Script/Gameplay/GameManager.cs [hunk 2]
```diff
@@ -236,27 +249,28 @@ private void Update()
                     double starPower = fiveFretPlayer.Engine.EngineStats.StarPowerAmount;
 
                     _debugText.text =
-                        $"Note index: {noteIndex}\n" +
-                        $"Buttons: {buttonMask}\n" +
-                        $"Star Power: {starPower:0.0000}\n" +
-                        $"TicksPerEight: {ticksPerEight}\n";
+                        $"<b>Player:</b> {fiveFretPlayer.Player.Profile.Name}\n" +
```
Comment (EliteAsian123) [inline Assets/Script/Gameplay/GameManager.cs:252]: The “Player” field only effects guitar players here, as the drum player one below has not been modified.

**Review by EliteAsian123 — CHANGES_REQUESTED** [review]

File: Assets/Script/Gameplay/GameManager.cs (hunk 1 above)
Comment (nevespt) [reply Assets/Script/Gameplay/GameManager.cs:147]: the idea was to not change the default, but if more the 1 i can see the board and not the taxt overlay it, thats why i change the layout in code, but just do it at the start

File: Assets/Script/Gameplay/GameManager.cs (hunk 2 above)
Comment (nevespt) [reply Assets/Script/Gameplay/GameManager.cs:252]: var PlayerName = _players[0].Player.Profile.Name;
this should work ig

File: Assets/Script/Gameplay/GameManager.cs (hunk 1 above)
Comment (RileyTheFox) [reply Assets/Script/Gameplay/GameManager.cs:147]: Yeah I have to agree, its debug text only meant for developer use or getting additional info from a player when reporting a bug. It's not meant to look pretty or do anything special, it serves one function and that is to provide debug info during gameplay.

**Pull request closed without merging**